cluster/gce/gci/configure-helper.sh

```diff
@@ -757,6 +824,23 @@ contexts:
   name: webhook
 EOF
   fi
+  if [[ "${ENABLE_KAS_NETWORK_PROXY:-false}" == "true" ]]; then
+    cat <<EOF >/etc/connectivity_service_configuration.yaml
```

Comment:
This should be in /etc/kubernetes or /etc/srv/kubernetes

Reply:
I think /etc/srv/kubernetes. Good call.